File: 7 kyu/You're a square!/solution.dart
import 'dart:math';

bool isSquare(num n) => sqrt(n) % 1 == 0;
__________________________________
import 'dart:math';

var isSquare = (n) => sqrt(n).remainder(1) == 0;
__________________________________
isSquare(n) { 
  for(var i = 0; i <= n; i++){
    if(i*i == n)
      return true;
  }
  return false;
}
__________________________________
import 'dart:math';
bool isSquare(int n) => n>=0 && sqrt(n) == sqrt(n).round();
__________________________________
import "dart:math";
bool isSquare(n) { 
var ra = sqrt(n);
return (ra%1==0) ?  true:  false;
  
}

File: 7 kyu/You're a square!/solution.cs
using System;

public class Kata
{
  public static bool IsSquare(int n)
  {
    return Math.Sqrt(n) % 1 == 0;
  }
}
__________________________________
using System;

public class Kata {
  public static bool IsSquare(int n) 
    => (Math.Sqrt(n) % 1 == 0);
}
__________________________________
using System;

public class Kata
{
  public static bool IsSquare(int n)
  {
    return Math.Abs(Math.Sqrt(n) - (int)Math.Sqrt(n)) < double.Epsilon;
  }
}
__________________________________
using System;

public class Kata
{
  public static bool IsSquare(int n)
  {
  bool sqr;
  double result;
  double result2;
    if (n >= 0)
    {
    result = Math.Sqrt(n);
    result2 = Math.Round(result);
    if (result2 == result)
       sqr = true;
    else
       sqr = false;
    }
    else
    {
    sqr = false;
    }
    return sqr;
  }
}
